third_party/terraform/resources/resource_google_project_service.go

```diff
@@ -84,19 +88,16 @@ func resourceGoogleProjectServiceRead(d *schema.ResourceData, meta interface{})
 	if err != nil {
 		return err
 	}
+	srv := d.Get("service").(string)
 
-	enabledServices, err := readEnabledServiceUsageProjectServices(project, config)
+	enabled, err := isServiceEnabled(project, srv, config)
```

Comment:
as a note, I don't think there was a single service.get method at the time this resource was implemented so we had to use list - let's not do that anymore :)